### PR TITLE
Block Fee Reward Calculation + other small things

### PR DESCRIPTION
**Pull request by randomshinichi**
1. Block fee reward calculation
2. list of commands on telnet is no longer a static string - it's a joined list of commands
3. an Exception can be requested over telnet (probably not that useful now)

### CODE REVIEW

File: walletprotocol.py [hunk 1]
```diff
@@ -229,6 +230,11 @@ def parse_cmd(self, data):
                 elif command == 'wallet':
                     self.wallet()
 
+                elif command == 'crash':
+                    self.output['message'].write('>>> CRASH requested' + '\r\n')
+                    print("==================CRASH REQUESTED OVER TELNET====================")
+                    raise Exception("CRASHING CRASHING CRASHING CRASHING CRASHING CRASHING CRASHING bail now")
```
Comment (jleni) [inline walletprotocol.py]: Although telnet should be removed at some point, adding a crash command looks like an odd idea..
What is the purpose/concept behind proposing a crash command?

**Comment by jleni** [comment]
HI @randomshinichi,
You should create pull request from branches. Otherwise every time you check-in something to your master the PR grows with distinct changes. Please group changes in different PRs/branches according to some theme. That will make easier to review them.

Could you also look at my question? I am not sure why we need a crash command in telnet. What is the concept behind that?

**Comment by randomshinichi** [comment]
OK, I'll create a new branch and cherry pick. cyyber told me to look into exception handling some time ago, so I needed a way to do that.

**Comment by jleni** [comment]
Ok but I don't think that should be in master (production code). 
With respect to this PR, I would recommend you to work only on branches and do not update them with additional changes after submitting a PR. It will make reviewing a lot easier.

**Comment by randomshinichi** [comment]
Hey @jleni, I'm making a new branch and PR. Should I leave out the telnet crash command?

**Pull request closed without merging**